.travis.yml

```diff
@@ -21,10 +22,11 @@ services:
 
 script:
 - |
-  if [[ "$TRAVIS_OS_NAME" == 'linux' && -n "$BASHVER" ]]; then
-    docker build --build-arg bashver=${BASHVER} --tag bats/bats:bash-${BASHVER} .
-    docker run -it bash:${BASHVER} --version
-    time docker run -it bats/bats:bash-${BASHVER} --tap /opt/bats/test
+  set -e
```

Comment:
Not a blocker for this PR, but we should consider re-working the travis config to use their new [conditions](https://docs.travis-ci.com/user/conditions-v1) branching